packages/upload-api/src/types/delegations.ts

```diff
@@ -14,6 +14,7 @@ export interface DelegationsStorage<
    * @param delegations - delegations to store
    */
   putMany: (
+    cause: Ucanto.Link,
     ...delegations: Array<Ucanto.Delegation<Ucanto.Tuple<Cap>>>
```

Comment:
We should just pass this as an array and probably of delegation CIDs as opposed to delegations.

> JS engines actually set limits on number of arguments function can have and I have experienced nasty bugs caused by going over that limit.

Reply:
I'm realizing that we need more details here than delegation CID, however it still probably better to pass records like `{ cause: CID, delegation: CID, audience: DID,  nbf?, exp? }` as opposed to whole Delegation.